conformance/packages/dns-test/src/container.rs

```diff
@@ -329,6 +362,10 @@ fn skip_docker_build() -> bool {
     env::var("DNS_TEST_SKIP_DOCKER_BUILD").is_ok()
 }
 
+fn docker_build_gha_cache() -> bool {
+    env::var("DNS_TEST_DOCKER_CACHE_GHA").is_ok()
```

Comment:
Maybe [`env!()`](https://doc.rust-lang.org/std/macro.env.html)?

Reply:
I think I'd rather keep it at run-time to keep it consistent with the other `DNS_TEST` environment variables.